job-dsls/jobs/prod/prod_kogito_properties_generator.groovy

```diff
@@ -5,7 +5,7 @@
 def kogitoProps ='''
 import groovy.json.JsonOutput
 
-node('kie-rhel7&&!master') {
```

Comment:
I would suggest to extract label expressions such as this one into a constant(s) so we can change it easily when we need to switch between the clouds (or in case the Jenkins plugin does not work as expected).

Reply:
done, good proposal! thanks